blazor/dropdown-list/accessibility.md

```diff
@@ -7,39 +7,44 @@ control: DropDown List
 documentation: ug
 ---
 
-# Accessibility in Blazor DropDown List Component
+# Accessibility in DropDown List 
 
 The DropDownList component has been designed, keeping in mind the `WAI-ARIA` specifications, and applied the WAI-ARIA roles, states, and properties along with `keyboard support`. This component is characterized by complete keyboard interaction support and ARIA accessibility support that makes it easy for people who use assistive technologies (AT) or those who completely rely on keyboard navigation.
 
 ## ARIA attributes
 
-The DropDownList component uses the `Listbox` role, and each list item has an `option` role. The following `ARIA attributes` denote the DropDownList state.
+The DropDownList component uses the `Combobox` role. The following `ARIA attributes` denote the DropDownList state.
 
 | **Properties** | **Functionalities** |
 | --- | --- |
-| aria-haspopup | Indicates whether the DropDownList input element has a popup list or not. |
-| aria-expanded | Indicates whether the popup list has expanded or not. |
-| aria-selected | Indicates the selected option. |
-| aria-readonly | Indicates the readonly state of the DropDownList element. |
-| aria-disabled | Indicates whether the DropDownList component is in a disabled state or not. |
-| aria-activedescendent | This attribute holds the ID of the active list item to focus its descendant child element. |
-| aria-owns | This attribute contains the ID of the popup list to indicate popup as a child element. |
+| [aria-haspopup](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-haspopup) | Indicates whether the DropDownList input element has a popup list or not. |
```

Comment:
don't link the Mozilla site, remove all links. Just highlight the aria attributes